packages/cryptography/src/Encryption/EncryptionConfig.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Tempest\Cryptography\Encryption;
+
+final class EncryptionConfig
+{
+    /**
+     * @param EncryptionAlgorithm $algorithm The algorithm used for encrypting and decrypting values.
+     * @param non-empty-string $key A private, secure encryption key.
+     */
+    public function __construct(
+        public EncryptionAlgorithm $algorithm,
+        #[\SensitiveParameter]
+        public string $key,
```

Comment:
Would it make sense to make this one readonly? Just to be absolutely sure the key cannot be changed within the context of a request or invocation?